### ISSUE TITLE
[🐛] Pods cant be installed

### ISSUE THREAD

**Issue by peyman-hakemi** [invertase/react-native-firebase#6836]
## Issue

After following the instructions for installing the firebase/app I can't install my pods
---

And I am getting this error

![image](https://user-images.githubusercontent.com/38393852/212668553-7d7ae149-7f42-4c09-b9f9-19037aebdf44.png)

I have tried adding a swift file to my project.

**Comment by AxraTech** [comment]
Same here

**Comment by evelant** [comment]
I think maybe there's somethign wrong with cocoapods in general? I haven't changed anything in my app but now it's failing to resolve several libraries when it built just fine a couple days ago.

```
[INSTALL_PODS] CocoaPods could not find compatible versions for pod "GoogleAppMeasurement":
[INSTALL_PODS]   In Podfile:
[INSTALL_PODS]     RNFBAnalytics (from `../node_modules/@react-native-firebase/analytics`) was resolved to 16.5.0, which depends on
[INSTALL_PODS]       Firebase/Analytics (= 10.3.0) was resolved to 10.3.0, which depends on
[INSTALL_PODS]         Firebase/Core (= 10.3.0) was resolved to 10.3.0, which depends on
[INSTALL_PODS]           FirebaseAnalytics (~> 10.3.0) was resolved to 10.3.0, which depends on
[INSTALL_PODS]             FirebaseAnalytics/AdIdSupport (= 10.3.0) was resolved to 10.3.0, which depends on
[INSTALL_PODS]               GoogleAppMeasurement (= 10.3.0)
[INSTALL_PODS]     RNGoogleMobileAds (from `../node_modules/react-native-google-mobile-ads`) was resolved to 8.2.0, which depends on
[INSTALL_PODS]       Google-Mobile-Ads-SDK (= 9.10.0) was resolved to 9.10.0, which depends on
[INSTALL_PODS]         GoogleAppMeasurement (< 10.0, >= 7.0)
[INSTALL_PODS]
```
The same thing is happening with `FBSDKCoreKit (~> 15.0.0)` and `CleverTap-iOS-SDK (= 4.1.5)`.

**Comment by mikehardy** [comment]
@peyman-hakemi with apologies, that is not a react-native-firebase issue, it is a problem with some unrelated pod
@evelant if you are using up to date react-native-google-mobile-ads there will be no version incompatibility, I work in that repo as well and remember we did an update many months back which brought things in line
